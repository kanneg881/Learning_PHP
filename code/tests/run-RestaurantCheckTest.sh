./vendor/bin/phpunit --bootstrap ./tests/RestaurantCheckTest.php
