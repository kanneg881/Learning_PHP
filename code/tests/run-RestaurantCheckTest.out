PHPUnit 7.2.4 by Sebastian Bergmann and contributors.

.                                                                   1 / 1 (100%)

Time: 24 ms, Memory: 4.00MB

OK (1 test, 1 assertion)
